ios/Framework/Core/Base/BaseListScreenlet/BaseListView.swift

```diff
@@ -19,7 +19,7 @@ public class BaseListView: BaseScreenletView {
 	
 	public static let DefaultSection = ""
 	
-	public var streamMode: Bool = false
+	internal var streamMode: Bool = false
```

Comment:
do you need there variables to store the same flag? what if they're inconsistent? can you do it with less variables? even with only one?